scripts/types.js

```diff
@@ -7,11 +7,25 @@ export const generateTypes = async (symbols, dir) => {
   const file = path.resolve(dir, 'index.d.ts');
   console.log(`Generating ${path.relative('', file)}`);
   const versions = await getVersions(symbols);
-  const types = getTypes(Object.keys(versions));
+  const types = getTypes(Object.keys(versions), symbols);
   await fs.writeFile(file, types);
   console.log('Done');
 };
 
-const getTypes = (icons) => {
-  return 'TODO';
+const getTypes = (icons, isSymbol) => {
```

Comment:
Nit: Please add `isSymbol` as first param. When symbols check is required in a function, I prefer to add it as a first param so that I don't accidentally miss it out e.g. calling `getTypes(Object.keys(versions))` would work but misses symbols.